collector/statsd/docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+version: "3.9"
+
+services:
+  statsd:
+    build:
+      context: .
+      dockerfile: Dockerfile.statsd
+    hostname: statsd
+  otel-collector:
+    image: otel/opentelemetry-collector-contrib:0.51.0
```

Comment:
@githubsands do we care about pulling the oldest vs early. Does it break for 0.56.0?